### ISSUE TITLE
a11y: Table block inverted style link color contrast too low

### ISSUE THREAD

**Issue by jackahl** [plone/volto#6334]
**Describe the bug**
When using the table block with inverted colors, the contrast between the bg and links is too low. We should choose a better blue here

**To Reproduce**

1. Go to https://volto.demo.plone.org/
2. Add a new table block
3. invert its color and add a inline link in on of the table cells
4. Use either axe or Colour contrast Analyser Software to check for sufficient contrast

--> contrast is not sufficient

**Expected behavior**
Contrast should be sufficient

**Screenshot**

![Screenshot 2024-09-26 at 11-29-10 table](https://github.com/user-attachments/assets/9a4fd030-9eb4-4d51-b048-1872581d6c23)

**Software:**

- OS: OSX (does not really matter)
- Browser: any
- Volto Version: latest main branch checkout
- Plone Version: 6

**Comment by pnicolli** [comment]
Fixed by #6333
